cli/smartcontract/contract_test.go

```diff
@@ -213,6 +213,110 @@ func ToMap(a []testcontract.MyPair) map[int]string {
 `, string(bs))
 }
 
+func TestDynamicWrapper(t *testing.T) {
+	smartcontract.ModVersion = "v0.0.0"
+
+	tmpDir := t.TempDir()
+	e := testcli.NewExecutor(t, true)
+
+	ctrPath := filepath.Join(tmpDir, "testcontract")
+	e.Run(t, "neo-go", "contract", "init", "--name", ctrPath)
+
+	cfgPath := filepath.Join(ctrPath, "neo-go.yml")
+	bindingsPath := filepath.Join(tmpDir, "bindings.yml")
+	manifestPath := filepath.Join(tmpDir, "manifest.json")
+	cmd := []string{"neo-go", "contract", "compile", "--in", ctrPath}
+
+	goMod := filepath.Join(ctrPath, "go.mod")
+	data, err := os.ReadFile(goMod)
+	require.NoError(t, err)
+
+	i := bytes.IndexByte(data, '\n')
+	data = append([]byte("module myimport.com/testcontract"), data[i:]...)
+
+	wd, err := os.Getwd()
+	require.NoError(t, err)
+	data = append(data, "\nreplace github.com/nspcc-dev/neo-go/pkg/interop => "...)
+	data = append(data, filepath.Join(wd, "../../pkg/interop")...)
+	require.NoError(t, os.WriteFile(goMod, data, os.ModePerm))
+
+	cmd = append(cmd, "--config", cfgPath,
+		"--out", filepath.Join(tmpDir, "out.nef"),
+		"--manifest", manifestPath,
+		"--bindings", bindingsPath)
+
+	e.Run(t, cmd...)
+	e.CheckEOF(t)
+	require.FileExists(t, bindingsPath)
```

Comment:
Not needed, see the comment above.

Reply:
Bindings generation step is missing. Generate contract binding for `verify.go` contract using `neo-go contract generate-wrapper` command and files from the comment above:
```
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ ./bin/neo-go contract generate-wrapper -h
NAME:
   neo-go contract generate-wrapper - generate wrapper to use in other contracts

USAGE:
   neo-go contract generate-wrapper --manifest <file.json> --out <file.go> --hash <hash> [--config <config>]

OPTIONS:
   --config value, -c value    Configuration file to use
   --manifest value, -m value  Read contract manifest (*.manifest.json) file
   --out value, -o value       Output of the compiled wrapper
   --hash value                Smart-contract hash

```